src/components/learnMore/components/LearnMore.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Icon } from 'antd';
+
+import { LEARN_MORE_CLASSNAMES, LERAN_MORE_DATA } from '../constants';
+
+const { HEADER, IMAGE, TEXT, BODY, ICON } = LEARN_MORE_CLASSNAMES;
+
+const learn_more = LERAN_MORE_DATA.map(({ key, image, header, body }) => (
```

Comment:
```suggestion
const learn_more
```
we use camelcase. change to `learnMore`